src/components/FridayOptions/index.jsx

```diff
@@ -1,37 +1,42 @@
-import { useState, useEffect, useCallback, useMemo } from 'react';
-import { DndProvider } from 'react-dnd';
-import { HTML5Backend } from 'react-dnd-html5-backend';
-import DraggableItem from './DraggableItem.jsx';
-import DropZone from './DropZone.jsx';
-import './style.css';
-import { options, zones } from './defaultOptions.json';
-import ArrowCircleLeftIcon from '@mui/icons-material/ArrowCircleLeft';
+import { useState, useEffect, useCallback, useMemo } from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import DraggableItem from "./DraggableItem.jsx";
+import DropZone from "./DropZone.jsx";
+import "./style.css";
+import { options, zones } from "./defaultOptions.json";
+import ArrowCircleLeftIcon from "@mui/icons-material/ArrowCircleLeft";
 
 const FridayOptions = ({ isVisible, closePanel }) => {
-  // Memoized available zones
-  const availableZones = useMemo(
-    () => zones.reduce((acc, zone) => ({ ...acc, [zone.type]: null }), {}),
-    [zones]
+  // derived state from zones
+  const availableZones = zones.reduce(
+    (acc, zone) => ({ ...acc, [zone.type]: null }),
+    {}
   );
 
   const [choices, setChoices] = useState(() => {
     // Initial load from localStorage
-    const data = localStorage.getItem('options');
-    return data ? JSON.parse(data) : availableZones;
+    const data = localStorage.getItem("options");
+    try {
+      return data ? JSON.parse(data) : availableZones;
+    } catch (error) {
+      console.error("Error parsing localStorage data:", error);
+      return availableZones;
+    }
```

Comment:
so, we do we need this `try - catch` structure:

when reading from local storage, this function: 

```js
JSON.parse(obj)
```

might raise an exception. So we need to catch, and define a "default branch" (`availableZones` in this case).

two options: keep this logic (as your original code) in the `useState()` initialisation.

```js
const [v, setV = useState(() => { .... })
```

option no. 2: **initial `useEffect()`**

```js
const [v, setV] = useState([])

useEffect(() => {
  // read logic
}, []) // this runs on component mount
```

any of the two options will work :)